docs/sources/articles/networking.md

```diff
@@ -121,8 +121,17 @@ Finally, several networking options can only be provided when calling
  *  `-P` or `--publish-all=true|false` — see
     [Binding container ports](#binding-ports)
 
-The following sections tackle all of the above topics in an order that
-moves roughly from simplest to most complex.
+For those networking options which you would like to supply to the Docker server when it starts up, you can set them up using `DOCKER_OPTS` in the Docker upstart configuration file. For ubuntu, edit the variable in /etc/default/docker while /etc/sysconfig/docker for centos. 
```

Comment:
The wrap should be to 80 characters. There are some English construction errors.  For an update see this gist:

https://gist.github.com/moxiegirl/a354e23c0cc9af0cd124